lib/cisco_node_utils/bgp.rb

```diff
@@ -360,6 +360,7 @@ def default_enforce_first_as
     # Nvgen as True With optional 'size <size>
     def event_history_cli
       match = config_get('bgp', 'event_history_cli', @get_args)
+      return default_event_history_cli if match == 'size_small'
```

Comment:
Right, which makes this code kind of ugly and a bit redundant where it returns the default in two places, however I wasn't able to find a better solution.